setup-environment-internal

```diff
@@ -237,6 +241,10 @@ if [ -d "${MANIFESTS}"/factory-keys ]; then
     if [ -d "${MANIFESTS}"/factory-keys/platform ] && [ ! -d "conf/factory-keys/platform" ]; then
         ln -sf "${MANIFESTS}"/factory-keys/platform conf/factory-keys/platform
     fi
+    # Link Composefs keys
+    if [ ! -d "conf/keys/cfs" ]; then
+        ln -sf "${MANIFESTS}"/conf/keys/cfs conf/keys/cfs
+    fi
```

Comment:
this snip need to be a bit different from the other
```
# Link Composefs keys if not set by the user
if [ -d "${MANIFESTS}"/factory-keys/cfs ] && [ ! -d "conf/factory-keys/cfs" ]; then
    ln -sf "${MANIFESTS}"/factory-keys/cfs conf/factory-keys/cfs
fi
```

Reply:
Yeah, link is incorrect.